extensions/amp-script/amp-script.md

```diff
@@ -138,6 +138,51 @@ Will be reflected on the page as a new child of the `amp-script` element:
 
 Under the hood, `amp-script` uses [@ampproject/worker-dom](https://github.com/ampproject/worker-dom/). For design details, see the ["Intent to Implement" issue](https://github.com/ampproject/amphtml/issues/13471).
 
+### Advanced usages
```

Comment:
Are there other "advanced usages" coming? Are we unsupportive of this use case or is this a common request?

Reply:
There will most likely be more "advanced usages" in the future. 

Right now, we think this is an interesting/unique capability of `amp-script` but it still needs more thought on how it can distill into concrete use cases.

Reply:
I vote for dropping "advanced usages" and titling this section as "State manipulation" or something similar.

Reply:
I wanted to highlight that it's kind of a niche use case since it requires the `amp-bind` extension to be installed too. There's already a lot of complex stuff in this documentation so being able to gloss over "Advanced usages" section might be useful. But I'll change it if you insist. :)

Reply:
I think advanced usages is a little misleading in that it makes it sound challenging rather than niche (although, is challenging misleading :P ?) and would make me believe it isn't a proper use case. 

This use also sounds like a great way to support full page manipulation without wrapping the entire page in `amp-script`. Which may not be so niche?

Reply:
Ok changed.